src/device-registry/routes/v1/activities.js

```diff
@@ -47,6 +47,17 @@ router.post(
         .notEmpty()
         .withMessage("the provided deviceName cannot be empty")
         .trim(),
+      body("recallType")
```

Comment:
**recallType** is now mandatory during device recall. So the frontend needs to be updated to bring in a popup or dropdown that requests a users to select what **type of recall** they will be doing. @BenjaminSsempala / @Codebmk